### ISSUE TITLE
[BUG]: Reported in Slack - Potential mis-assignment of error to 2.4.3 Focus Order rather than 2.1.1 Keyboard

### ISSUE THREAD

**Issue by maryjom** [IBMa/equal-access#1834]
### Project

Checker engine - rule mappings

### Browser

_All_

### Operating system

_All_

### Description

~~Don't know the project, browser, or OS as this is an issue based on Slack question regarding a particular rule's error message and help.~~
#### Rules affected:  
**_widget_tabbable_exists_**
**_widget_tabbable_single_**

It appears that the rule may be citing an incorrect WCAG success criterion for the particular message it provides (a mismatch). The following is my Slack message which contains a link to the original user's question:  

>In looking at [this question](https://ibm-studios.slack.com/archives/C036P1CTN/p1706888064103939) posed in [#accessibility-at-ibm](https://ibm-studios.slack.com/archives/C036P1CTN), I'm wondering why this error is being reported against 2.4.3 Focus Order rather than 2.1.1 Keyboard.  Seems if a tablist doesn't have an element in the tab order at all, it should have one and it would fail 2.1.1 Keyboard. Then you'd have to manually check that the tab/focus order is correct within the tablist for 2.4.3.

### Steps to reproduce

Contact the originator of the question in the Slack #accessibility-at-ibm channel to see if they can provide code for you to reproduce the error. 

or simply:
1. Run the test cases for the rules and check the mapping
2. Review the mapping in the [Checker rule set](https://www.ibm.com/able/requirements/checker-rule-sets/) for the rules mapped to 2.1.1 and 2..4.3.  The mapping is also available in the Actions -> Artifacts: Rule listing (generate are runtime)

**Comment by philljenkins** [comment]
Current mapping:
#### 2.4.3 Focus Order (A)
If content can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability.
- **_widget_tabbable_exists_**: Component must have at least one tabbable element
 Needs review
 [Component with "{0}" role does not have a tabbable element](https://unpkg.com/accessibility-checker-engine@3.1.67/help/en-US/widget_tabbable_exists.html#%7B%22message%22%3A%22Component%20with%20%5C%22%7B0%7D%5C%22%20role%20does%20not%20have%20a%20tabbable%20element%22%2C%22msgArgs%22%3A%5B%22%7B0%7D%22%2C%22%7B1%7D%22%2C%22%7B3%7D%22%2C%22%7B4%7D%22%5D%2C%22value%22%3A%5B%22VIOLATION%22%2C%22POTENTIAL%22%5D%2C%22reasonId%22%3A%22fail_no_tabbable%22%7D)
- **_widget_tabbable_single_**: Certain components must have no more than one tabbable element
 Needs review
 [Component with "{0}" role has more than one tabbable element](https://unpkg.com/accessibility-checker-engine@3.1.67/help/en-US/widget_tabbable_single.html#%7B%22message%22%3A%22Component%20with%20%5C%22%7B0%7D%5C%22%20role%20has%20more%20than%20one%20tabbable%20element%22%2C%22msgArgs%22%3A%5B%22%7B0%7D%22%2C%22%7B1%7D%22%2C%22%7B3%7D%22%2C%22%7B4%7D%22%5D%2C%22value%22%3A%5B%22VIOLATION%22%2C%22POTENTIAL%22%5D%2C%22reasonId%22%3A%22fail_multiple_tabbable%22%7D)

**Comment by philljenkins** [comment]
Current Help:
#### About this requirement
- [IBM 2.1.1 Keyboard](https://www.ibm.com/able/requirements/requirements/#2_1_1)
- [ARIA practices - Developing a Keyboard Interface](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/)
- [ARIA specification - Widget Roles](https://www.w3.org/TR/2023/REC-wai-aria-1.2-20230606/#widget_roles)

**Comment by shunguoy** [comment]
Triage: this mapping is related to KCM for sequential navigation of the browser extension.  
   widget_tabbable_single: map to 2.4.3 and 2.1.1
    widget_tabbable_exists: map to just 2.1.1

**Comment by philljenkins** [comment]
1. Consider adding the following "toolkit testing" reference to the help
[Check tab or navigation order](https://www.ibm.com/able/toolkit/verify/manual/#tabnavorder)
https://www.ibm.com/able/toolkit/verify/manual/#tabnavorder

2. Update the style & text of the group message of

**Comment by philljenkins** [comment]
prioritize lowest in release

**Comment by marcjohlic** [comment]
Tackle this one AFTER scheduled Release items are completed.   We don't want this to impede any of the Release items.

**Comment by philljenkins** [comment]
Rescheduling this to next sprint to continue working on AccessU outline

**Comment by shunguoy** [comment]
branch: dev-1834

**Comment by shunguoy** [comment]
PR: https://github.com/IBMa/equal-access/pull/1899

**Comment by shunguoy** [comment]
PR: https://github.com/IBMa/equal-access/pull/1899

**Comment by philljenkins** [comment]
Changes made to mapping and help "Why is this important" and "About this requirement" references
